src/core/logic.py

```diff
@@ -132,20 +126,10 @@ def book_to_mark21_file(book, owner, xml=False):
             )
         )
 
-    try:  # Publication.
-        press_name = models.Setting.objects.get(
-            group__name='general',
-            name='press_name',
-        ).value
-    except:
-        press_name = None
-    try:
-        city = models.Setting.objects.get(
-            group__name='general',
-            name='city'
-        ).value
-    except:
-        city = None
+    # Publication.
+    press_name = get_setting('press_name', 'general')
+
+    city = get_setting('city', 'general')
```

Comment:
Will this return `None` as well?

Reply:
Yes, that's the default value of `default`.

Reply:
Sorry, ignore the previous comment. Although the default for the `default` argument is `None`, the get_setting function will return and empty string, rather than none, if the default is `None`.

Reply:
As the values of `press_name` and `city` are treated as booleans below, and empty strings would evaluate False, it should be fine.